docker-compose.yml

```diff
@@ -126,7 +126,8 @@ services:
     env_file:
       - .env.nginx.local
     healthcheck:
-      test: curl -s -o /dev/null -w "%{http_code}" "http://localhost/nginx-health" | grep -q "^200$" || false
+      test: |-
+        curl -s -o /dev/null -w "%{http_code}" --header 'Host: davidrunger.com' http://localhost/nginx-health | grep -q "^200$" || false
```

Comment:
Manually providing a `Host` header is a way to circumvent our efforts to require requests to go through Cloudflare (via DNS resolution), rather than permitting requests made directly to the server's IP address. However, we can circumvent that protection, as seen here, and hackers can do this, too (and, from some log lines that I have seen, I believe that they actually have done this and been noticed by me, which somewhat surprises me).